ClubMembersSortedCF/src/members/v1/Validator.java

```diff
@@ -0,0 +1,5 @@
+package members.v1;
+
+public abstract class Validator<T> {
+    abstract boolean isValid(T t);
```

Comment:
please note that if this was an interface, using `no visibility` modifier would imply that the given method is public; here it is package-private instead. (i have no issue here, just wanted to note this)